Todo.py

```diff
@@ -0,0 +1,203 @@
+                            repostMessage = await bot.get_channel(int(repostFieldValue.split(':')[0][2:-2])).fetch_message(int(repostFieldValue.split(':')[1][1:]))
+                            await repostMessage.delete()
+                    elif reaction.emoji.name == EMOJIS['check']:
+                        await message.remove_reaction(EMOJIS['hourglass'], bot.user)
+                    elif reaction.emoji.name == EMOJIS['hourglass']:
+                        await message.remove_reaction(EMOJIS['check'], bot.user)
+
+    @commands.Cog.listener()
+    async def on_raw_reaction_remove(self, reaction):
+        message = await bot.get_channel(reaction.channel_id).fetch_message(reaction.message_id)
+
+        with open(TODO_CHANNEL, "r") as file:
+            channelId = file.readlines()[0]
+
+        if reaction.channel_id == int(channelId): # Check if it's a todo-message (check if it's the good channel)
+            if len(message.embeds) > 0: # Check if it's an embed, I think this will avoid most problems
+                if reaction.user_id != bot.user.id:
+                    if reaction.emoji.name == EMOJIS['check']:
+                        await message.add_reaction(EMOJIS['hourglass'])
+                    elif reaction.emoji.name == EMOJIS['hourglass']:
+                        await message.add_reaction(EMOJIS['check'])
+
+    @commands.group()
+    async def todo(self, ctx):
+        '''Commands to manage a todolist.'''
+        if ctx.invoked_subcommand is None:
+            await ctx.send('Error: See for ``' + PREFIX + 'help todo``')
+
+    @todo.command()
+    async def add(self, ctx, *args):
+        '''Command to add a todo. Usage : <the thing todo>;<type of todo>;<assigned to / false>;<repost public channel / false>'''
+        with open(TODO_CHANNEL, "r") as file:
+            channelId = file.readlines()[0]
+            if channelId == None or channelId == "":
+                await ctx.channel.send("The todos' channel must be selected with the command " + PREFIX + "todo channel")
+                return
+        channel = bot.get_channel(int(channelId))
+
+        command = ""
+        for arg in args:
+            command += " {}".format(arg)
+        command = command.split(";")
+
+        todoType = None
+
+        with open(TODO_TYPES, "r") as file:
+            lines = file.readlines()
+            for line in lines:
+                line = line.split(';')
+                if command[1] == line[0]:
+                    todoType = line
+                    break
+        
+        if todoType != None:
+            embedColor = int(todoType[1], 16)
+        else:
+            embedColor = 0x28a745
+        
+        newEmbed = discord.Embed(description=command[0], url="", color=embedColor)
+        
+        command[2] = command[2].replace(' ', '') #TODO: Use dfind instead ?
+        if command[2] != "false":
+            if command[2].startswith("<@"):
+                user = ctx.guild.get_member(int(command[2][2:-1]))
```

Comment:
Maybe you could use a `discord.py` converter here ? Not sure if it'd be better, I'll let you judge about it but keep me informed ;)

Reply:
How do you do ? ^^

Reply:
Look at the doc :P